api_formatter/templates/rest_framework/api.html

```diff
@@ -162,12 +164,12 @@ <h4>Limiting available</h4>
     {% endif %}
 
     <div class="request-info" style="clear: both" aria-label="{% trans "request info" %}">
-      <pre class="prettyprint"><b>{{ request.method }}</b> {{ request.get_full_path }}</pre>
+      <pre class="prettyprint"><span class=font-weight-bold>{{ request.method }}</span> {{ request.get_full_path }}</pre>
```

Comment:
I know it works without but let's get this class name in quotes?

Reply:
good catch!